src/layer/utils.ts

```diff
@@ -140,17 +145,33 @@ export function parseSHInstanceId(baseUrl: string): string {
   throw new Error(`Could not parse instanceId from URL: ${baseUrl}`);
 }
 
+export function getConfigurationServiceHostFromBaseUrl(baseUrl: string): string {
+  let host = baseUrl;
+
+  if (/\ogc\/wms/.test(baseUrl)) {
+    host = baseUrl.substring(0, baseUrl.indexOf('/ogc/wms') + 1);
+  }
+
+  if (/dataspace.copernicus.eu/.test(host)) {
+    return host;
+  }
+
+  // Note that for SH v3 service, the endpoint for fetching the list of layers is always
```

Comment:
Since this comment is not correct in general anymore we should perhaps mention the exception of Copernicus endpoint.